### PR TITLE
Adding soft body character challenge.

### PR DESCRIPTION
**Pull request by kfahn22**

### CODE REVIEW

File: content/videos/challenges/177-soft-body-character/index.json [hunk 1]
```diff
@@ -0,0 +1,192 @@
+{
+    "title": "Soft Body Character",
+    "description": "",
+    "videoId": "PKrvQLbyy08&t",
+    "nebulaSlug": "codingtrain-coding-challenge-soft-body",
+    "date": "2023-04-??",
```
Comment (fturmel) [inline content/videos/challenges/177-soft-body-character/index.json:6]: Don't forget to fix the date!

**Comment by fturmel** [comment]
@shiffman heads up, the YouTube video is still members-only

Screenshot 2023-04-13 at 10 59 49 AM

**Comment by shiffman** [comment]
Yeah, this is by design, I don't always do this, but there are still some little details to finish and I want to give members a chance to see it first!

**Comment by fturmel** [comment]
> Yeah, this is by design, I don't always do this, but there are still some little details to finish and I want to give members a chance to see it first!

Makes sense, I thought it might've been a mistake since you merged the PR already, and it's now public on thecodingtrain.com

**Comment by shiffman** [comment]
Yes, thanks for the feedback, I should probably develop a more clear and consistent process!

**Pull request merged**